### DISCUSSION TITLE
Contribution with runners

### DISCUSSION THREAD

**Discussion by EduardTheFirst** [lutris/lutris#3694] [Show and tell]
Hello to the developers,

I want to contribute by adding and configuring more runners 
for use in lutris
i tried to play with it on my machine(linux) and edit the configuration files
and i could not start a game because or to add the runner to the runners category

please guide to this little steps so i could make more readme files for contributing to this project :)

**Comment by assist25** [comment]
i also think the same thing. But nothing is open for this contribution. I'm already planning. To make a cap32 runner but still limited to engines that are in the project. Please understand
